docs/product/dev-toolbar/index.mdx

```diff
@@ -0,0 +1,52 @@
+
+The issues panel shows you your highest priority frontend [issues](https://docs.sentry.io/product/issues/) for the page you are currently viewing. From the Dev Toolbar, you can quickly jump into specific issues in Sentry to get more detailed information and take action (for example, assign to a team member or mark as resolved). With the issues panel, you can browse the pages that are most important to your business and understand the top issues impacting your user base.
+
+![Dev Toolbar with the Issues Panel open](./img/issues-panel.png)
+
+## Feedback Panel
+
+The feedback panel shows you the most recent user feedback messages for the page you are on, so you can more easily contextualize user feedback. The feedback can  include misleading UX, broken links, typos, and so on, covering more bugs and suboptimal experiences than code-thrown errors. The feedback panel works by pulling feedback messages from [Sentry’s User Feedback Widget](https://docs.sentry.io/product/user-feedback/#user-feedback-widget).
+
+![Dev Toolbar with the Feedback Panel open](./img/feedback-panel.png)
+
+## Feature Flags Panel
+
+Feature flags are a powerful tool that allow you to control the visibility of features in your app, enabling you to ship, test, and experiment with confidence. The feature flag panel allows you to quickly view and override feature flags locally that your team has enabled for your web application. You can override any feature flag to be ‘true’ or ‘false’ for your browser session, so you can verify its behavior and observe the impact it might have on errors. Learn more about how to [configure the feature flag panel](/product/dev-toolbar/setup/#implement-feature-flag-adapter).
+
+[Learn more](/product/explore/feature-flags/) about what data Sentry can track related to your feature flagging system.
+
+![Dev Toolbar with the Feature Flags Panel open](./img/feature-flags-panel.png)
+
+## Login Flow
+
+After setting up the Dev Toolbar you’ll see a “Login to Sentry” button floating in the center of the page. Your team will need to click this button and login to sentry before they can access any information from your Sentry organization.
+![Dev Toolbar login button](./img/login-button.png)
+
+After clicking the button, you should see a pop up for logging in to Sentry.
+![Sentry log in pop up](./img/login-popup.png)
+
+If it's your first time setting up the Dev Toolbar, there will be a button with a link to Sentry's Dev Toolbar settings page to configure your domain.
+![Dev Toolbar configure domains button](./img/configure-domain.png)
+
+You'll need to add your domain URL to Allowed Origins on Sentry's Dev Toolbar settings page.
+![Sentry's Dev Toolbar Settings Page](./img/sentry-toolbar-settings.png)
+
+Learn more about [deploying the toolbar to different environments](/product/dev-toolbar/setup/#deploying-to-production--dev-environments), and [conditionally inserting the toolbar script tag](/product/dev-toolbar/setup/#conditionally-inserting-script-tag).
```

Comment:
These links go to the bottom sections of the next page, which seems weird. Why not the top?

It looks like this, there is is a link "Next: Set Up"
![SCR-20241212-omak](https://github.com/user-attachments/assets/002fc3f1-b119-48ce-982e-84a1cb713b12)



I feel like, we should link people to the top of the next page, and use the sentence here as a teaser for what sections are on that page... 

> 

```suggestion
Learn more about [setting up the toolbar](/project/-dev-toolbar/setup/) including configuring a feature flag provider and including the toolbar in different environments.
```

maybe with all the links... but def the first link to the top of the page.

Reply:
Do we have a reason to believe people don't click on "Next Set Up >>" and that's why we are adding the link to this sentence? I think we shouldn't have any link because then I would be wondering if the link in the sentence takes me to the same place as the button.

```suggestion
Learn more about setting up the toolbar including configuring a feature flag provider and including the toolbar in different environments on the next page.
```